### PR TITLE
Optional labels

### PR DESCRIPTION
**Pull request by jprider63**
Before submitting your PR, check that you've:

- [x] Bumped the version number

After submitting your PR:

- [ ] Update the Changelog.md file with a link to your PR
- [ ] Check that CI passes (or if it fails, for reasons unrelated to your change, like CI timeouts)

### CODE REVIEW

**Comment by jprider63** [comment]
Actually, I've only tested this with `BootstrapBasicForm`. It might not make sense for `BootstrapHorizontalForm`.

**Comment by snoyberg** [comment]
Did you want to investigate that before I review?

**Comment by jprider63** [comment]
Sure. I should be able to later this afternoon.

**Comment by jprider63** [comment]
According to the [documentation](https://getbootstrap.com/docs/3.3/css/#forms-inline), it looks like we should use `.sr-only` to hide labels so I've switched to do this. Inline forms currently always have `.sr-only` so I left it as is.

File: yesod-form/ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # ChangeLog for yesod-form
 
+## 1.6.5
+
+* Omit labels in `renderBootstrap3` when they are null.
```
Comment (snoyberg) [inline yesod-form/ChangeLog.md]: Perhaps update this to clarify we use `.sr-only`?

**Pull request merged**